### PR TITLE
sys-process/iotop: delayacct issue

### PR DESCRIPTION
**Pull request by samuelbernardo**
Add post install instructions to add delayacct into kernel parameters.

Closes: #833617

### CODE REVIEW

**Comment by samuelbernardo** [comment]
> The format is: `Closes: https://bugs.gentoo.org/num`.

Just corrected commit message. Hope everything ok now.

**Comment by thesamesam** [comment]
Almost -- no need for both Bug and Closes. Closes is stronger than Bug, so just do that :)

File: sys-process/iotop/iotop-0.6.ebuild [hunk 1]
```diff
@@ -33,3 +33,15 @@ PATCHES=(
 pkg_setup() {
 	linux-info_pkg_setup
 }
+
+pkg_postinst() {
+	cat << EOF | elog
+To enable the SWAPIN/IO columns Edit /etc/default/grub,
+adding "delayacct" as an option to the GRUB_CMDLINE_LINUX_DEFAULT
```
Comment (Flowdalic) [inline sys-process/iotop/iotop-0.6.ebuild:40]: grub is only one way to boot the Linux kernel. I think the first message should be "To enable the SWAPIN/IO columns, set the 'delayacct' kernel command-line parameter". Then you can go on with: "For example if using grub, set or append GRUB_CMDLINE_LINUX_DEFAULT with 'delayacct' in /etc/default/grub". Maybe a reference to the kernel's documentation on delay accounting can't hurt too: https://www.kernel.org/doc/html/latest/accounting/delay-accounting.html

Comment (samuelbernardo) [reply sys-process/iotop/iotop-0.6.ebuild:40]: @Flowdalic Thanks for the review... I'm going update the PR with your suggestions.

Comment (samuelbernardo) [reply sys-process/iotop/iotop-0.6.ebuild]: Done.
I think thjis should be ready.

**Review by Flowdalic — CHANGES_REQUESTED** [review]
Looks good, but please fix the QA issues (trailing whitespace).

**Pull request closed without merging**